examples/src/bin/rpk_example.rs

```diff
@@ -0,0 +1,390 @@
+use log::trace;
```

Comment:
It would also be nice if the `.github/workflows/daily-tests.yml` could be updated to include some coverage in the `example-tests` job for this new example (assuming its not too painful).